.github/workflows/nightly_merge_master.yml

```diff
@@ -6,6 +6,8 @@ on:
   push:
     branches:
       - master
+  schedule:
+    - cron: '30 */3 * * *'  # 8AM CET/11PM PT
```

Comment:
Don't you also have to remove the `push` event?

Reply:
🤦‍♀️  Thanks.